beacon-chain/blockchain/forkchoice/process_attestation.go

```diff
@@ -93,15 +96,30 @@ func (s *Store) OnAttestation(ctx context.Context, a *ethpb.Attestation) (uint64
 		return 0, err
 	}
 
-	// Use the target state to to validate attestation and calculate the committees.
-	indexedAtt, err := s.verifyAttestation(ctx, baseState, a)
-	if err != nil {
-		return 0, err
-	}
+	// Process aggregated attestation in the queue every `slot/2` duration,
+	// this allows the incoming attestation to aggregate and avoid
+	// processing individual attestations.
+	if uint64(time.Now().Unix())%params.BeaconConfig().SecondsPerSlot/2 == 0 {
```

Comment:
This if is a little unwieldy, maybe we can abstract it into a small helper func at the bottom of the file